src/api/status/public/index.html

```diff
@@ -1178,6 +1077,6 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <!-- Control Center for Material Dashboard: parallax effects, scripts for the example pages etc -->
     <script src="assets/js/material-dashboard.min.js?v=3.0.0"></script>
     <!-- Telescope JS Files -->
-    <script type="module" src="js/pages/dashboard.js"></script>
+    <script src="js/github-stats.js"></script>
```

Comment:
`js/github-stats.js` itself doesn't do anything. It needs to attach to the load event inside `js/pages/dashboard.js`.

Reply:
Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it